pkg/controller/flink/job_manager_controller.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/lyft/flytestdlib/promutils/labeled"
 	v1 "k8s.io/api/apps/v1"
 	coreV1 "k8s.io/api/core/v1"
-	k8_err "k8s.io/apimachinery/pkg/api/errors"
+	k8err "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
This is a minor style thing suggested by my editor. So I went ahead and fixed it. Snake case variable names are discouraged in Go.